src/mon/FSCommands.cc

```diff
@@ -322,11 +322,13 @@ class SetHandler : public FileSystemCommandHandler
 
   int handle(
       Monitor *mon,
-      FSMap &fsmap,
+      FSMap &f,
       MonOpRequestRef op,
       const cmdmap_t& cmdmap,
       std::stringstream &ss) override
   {
+    FSMap fsmap = f;
```

Comment:
I'm not thrilled about the implicit `FSMap` copies that are going to be everywhere. Can you think iof a way to hide this in teh `FileSystemCommandHandler`. It also could become expensive and hard to follow.

Reply:
Right, I'm on it.

Reply:
Fixed.